synapse/rest/client/v1/_base.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Matrix.org Foundation C.I.C
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from synapse.rest.client.v2_alpha._base import client_patterns as _client_patterns
+
+
+def client_patterns(*args, **kwargs):
```

Comment:
Can we please call this something different as it does a different thing (by default) than the *other* client_patterns, e.g. `client_patterns_with_v1`? Or alternatively move the `v2_alpha` version up here and specifically add `v1=True` to the places that need it.